mysite/blog/templates/blog/base.html

```diff
@@ -0,0 +1,18 @@
+{% load static %}
```

Comment:
Use formatter for html files. It is unreadable without indentation.

Reply:
Formatted the HTML files